apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -241,17 +241,21 @@ export class SendMessageInApp extends SendMessageBase {
     }
 
     if (oldMessage) {
-      await this.messageRepository.update(
+      message = await this.messageRepository.findOneAndUpdate(
         { _environmentId: command.environmentId, _id: oldMessage._id },
         {
           $set: {
             seen: false,
             createdAt: new Date(),
+            updatedAt: new Date(),
             ...channelData,
           },
+        },
+        {
+          timestamps: false,
+          strict: false,
```

Comment:
There was a bug here, in order to update `createdAt` (for the message to pop on top of inbox), there need to be additional options to allow overwriting immutable fields.